tests/test_histEFT_add.py

```diff
@@ -0,0 +1,299 @@
+    assert baa.integrate("type", "non-eft").view(as_dict=True)[()][0][0] == nevts
+
+
+def test_add_abb():
+    ab = a + b
+    abb = ab + b
+    assert np.all(abb.integrate("type", "eft").view(as_dict=True)[()][0] == sums)
+    assert abb.integrate("type", "non-eft").view(as_dict=True)[()][0][0] == 2 * nevts
+
+
+def test_add_bab():
+    ba = b + a
+    bab = ba + b
+    assert np.all(bab.integrate("type", "eft").view(as_dict=True)[()][0] == sums)
+    assert bab.integrate("type", "non-eft").view(as_dict=True)[()][0][0] == 2 * nevts
+
+
+def test_add_ab_weights():
+    ab = a_w + b_w
+    assert np.all(
+        np.abs(ab.integrate("type", "eft").view(flow=False)[()][0] - weight_val * sums)
+        < 1e-10
+    )
+
+    assert np.all(
+        np.abs(
+            ab.integrate("type", "non-eft").view(flow=False)[()][0][0]
+            - weight_val * nevts
+        )
+        < 1e-10
+    )
+
+
+def test_add_ba_weights():
+    ba = b_w + a_w
+    assert np.all(
+        np.abs(ba.integrate("type", "eft").view(flow=False)[()][0] - weight_val * sums)
+        < 1e-10
+    )
+    assert np.all(
+        np.abs(
+            ba.integrate("type", "non-eft").view(flow=False)[()][0][0]
+            - weight_val * nevts
+        )
+        < 1e-10
+    )
+
+
+def test_add_aba_weights():
+    ab = a_w + b_w
+    aba = ab + a_w
+    assert np.all(
+        np.abs(
+            aba.integrate("type", "eft").view(flow=False)[()][0] - 2 * weight_val * sums
+        )
+        < 1e-10
+    )
+    assert np.all(
+        np.abs(
+            aba.integrate("type", "non-eft").view(flow=False)[()][0][0]
+            - weight_val * nevts
+        )
+        < 1e-10
+    )
+
+
+def test_add_baa_weights():
+    ba = b_w + a_w
+    baa = ba + a_w
+    assert np.all(
+        np.abs(
+            baa.integrate("type", "eft").view(flow=False)[()][0] - 2 * weight_val * sums
+        )
+        < 1e-10
+    )
+    assert np.all(
+        np.abs(
+            baa.integrate("type", "non-eft").view(flow=False)[()][0][0]
+            - weight_val * nevts
+        )
+        < 1e-10
+    )
+
+
+def test_add_abb_weights():
+    ab = a_w + b_w
+    abb = ab + b_w
+    assert np.all(
+        np.abs(abb.integrate("type", "eft").view(flow=False)[()][0] - weight_val * sums)
+        < 1e-10
+    )
+    assert np.all(
+        np.abs(
+            abb.integrate("type", "non-eft").view(flow=False)[()][0][0]
+            - 2 * weight_val * nevts
+        )
+        < 1e-10
+    )
+
+
+def test_add_bab_weights():
+    ba = b_w + a_w
+    bab = ba + b_w
+    assert np.all(
+        np.abs(bab.integrate("type", "eft").view(flow=False)[()][0] - weight_val * sums)
+        < 1e-10
+    )
+
+    assert np.all(
+        np.abs(
+            bab.integrate("type", "non-eft").view(flow=False)[()][0][0]
+            - 2 * weight_val * nevts
+        )
+        < 1e-10
+    )
+
+
+def test_split_by_terms():
+    # split_by_terms not yet implemented
+    raise NotImplementedError
```

Comment:
Should  we remove this from the test? The CI will run `pytest` which runs all of these.

Reply:
I added that because the new hist does not have a split_by_terms yet. Do you mean to remove the test or the raised error? Without that raise, the test fails with an attribute error.  Maybe replace with something like: 

```python
@pytest.mark.skipif(getattr(HistEFT, "split_by_terms", None) is None)
def test_split_by_terms():
  ... etc
```
?

Reply:
I think we should remove the test (if you just remove `test` from the name, `pytest` will skip it). What do you think @kmohrman?

Reply:
I guess we just want to avoid the CI failing due to known things. Does the current implementation cause the CI to fail?